src/notification.c

```diff
@@ -223,10 +225,12 @@ fill_background(GtkWidget *widget, WindowData *windata, cairo_t *cr) {
     cairo_fill_preserve (cr);
 
     if (windata->settings.border) {
-        r = (float) color.red / 65535.0;
-        g = (float) color.green / 65535.0;
-        b = (float) color.blue / 65535.0;
-        cairo_set_source_rgba (cr, 0, 0, 0, windata->settings.alpha);
+        GdkColor color1;
```

Comment:
`color1` is not very descriptive. Could be renamed to something like `border_color`

Reply:
My bad, as I can see the `color` variable above can be reused, since the only thing it is used is for getting r,g,b values. Is that ok, if I reuse `color` variable or should I rename `color1` to `border_color`?

Reply:
I believe its fine if you want to reuse it

Reply:
The `GdkColor color` is now reused